tests/README.tests

```diff
@@ -0,0 +1,6 @@
+To run tests:
+
```

Comment:
Could you also leave instructions on how to create a proper dev environment? Here is my attempt.

```
Setting up the dev environment
---------------------------------
To isolate dependencies from your global python installation, it is important to use a tool like
[virtualenv](https://virtualenv.pypa.io/en/stable/). With `virtualenv` you can install the dev environment by doing the following.

- `pip install -e .`
- `pip install -r dev-requirements.txt`